docs/components_page/components/carousel/captions.py

```diff
@@ -0,0 +1,31 @@
+import dash_bootstrap_components as dbc
+
+carousel = dbc.Carousel(
+    items=[
+        {
+            "key": "1",
+            "src": "/static/images/slide1.svg",
+            "header": "With header ",
+            "caption": "and caption",
+        },
+        {
+            "key": "2",
+            "src": "/static/images/slide2.svg",
+            "header": "With header only",
+            "caption": "",
+        },
+        {
+            "key": "3",
+            "src": "/static/images/slide3.svg",
+            "header": "",
+            "caption": "This slide has a caption only",
+        },
+        {
+            "key": "4",
+            "img_style": {"height": "300px"},
+            "header": "This slide has no image",
+            "caption": "Image height must be set to make the slide visible",
+            "captionClassName": "border border-secondary text-primary",
+        },
```

Comment:
This looks a bit strange when I run it, is that expected?

![image](https://user-images.githubusercontent.com/15220906/124344519-a7cc6880-dbca-11eb-9223-15e71e40e6ff.png)

Reply:
Yeah, this does look a little strange.  I'm not sure if it's even necessary to show how to do a slide without an image - that's not included in the official Bootstrap docs for Carousel.  (But I figured out how to do it, so I thought it would be good to include as an example)

For this use-case, it would probably be better to save any text you want displayed as an image, then include it as a slide.  Should I just delete this 4th slide?

Reply:
Yeah, I think just delete it. Not sure it would be a particularly common use case